tests/test_stacks.py

```diff
@@ -576,31 +576,41 @@ def test_integrated_stack_endpoint_simple(client, phabdouble, mocked_repo_config
     r2 = phabdouble.revision(repo=repo, depends_on=[r1])
     r3 = phabdouble.revision(repo=repo, depends_on=[r1])
     r4 = phabdouble.revision(repo=unsupported_repo, depends_on=[r2, r3])
+    r5 = phabdouble.revision(repo=None, depends_on=[r1])
 
     response = client.get("/stacks/D{}".format(r3["id"]))
     assert response.status_code == 200
 
-    assert len(response.json["edges"]) == 4
+    assert len(response.json["edges"]) == 5
     assert [r2["phid"], r1["phid"]] in response.json["edges"]
     assert [r3["phid"], r1["phid"]] in response.json["edges"]
     assert [r4["phid"], r2["phid"]] in response.json["edges"]
     assert [r4["phid"], r3["phid"]] in response.json["edges"]
+    assert [r5["phid"], r1["phid"]] in response.json["edges"]
 
     assert len(response.json["landable_paths"]) == 2
     assert [r1["phid"], r2["phid"]] in response.json["landable_paths"]
     assert [r1["phid"], r3["phid"]] in response.json["landable_paths"]
 
-    assert len(response.json["revisions"]) == 4
+    assert len(response.json["revisions"]) == 5
     revisions = {r["phid"]: r for r in response.json["revisions"]}
     assert r1["phid"] in revisions
     assert r2["phid"] in revisions
     assert r3["phid"] in revisions
     assert r4["phid"] in revisions
+    assert r5["phid"] in revisions
 
     assert revisions[r4["phid"]]["blocked_reason"] == (
         "Repository is not supported by Lando."
     )
 
+    assert revisions[r5["phid"]]["blocked_reason"] == (
```

Comment:
This change is testable from a unit test, I think I'd prefer if we had something mimicking one of the `test_calculate_landable_subgraphs_*` tests, specifically for this case, rather than making this test more complicated. Because they only rely on the `phabdouble` fixture, they're much much cheaper than anything relying on `client` etc. (which requires things like the database).